### PR TITLE
[REVIEW] Updating order of function arguments in _kneighbors call to cuml.

### PR DESCRIPTION
**Pull request by cjnolet**
These function arguments were re-arranged in a recent change to cuML's `_kneighbors` function. This PR updates NearestNeighbors.

### CODE REVIEW

File: dask_cuml/neighbors/nearest_neighbors.py [hunk 1]
```diff
@@ -200,8 +200,8 @@ def collect_ipcs(ipcs):
 
     for idx, allocs in alloc_info:
         X, inds, dists = allocs
-        m._kneighbors(X["data"][0], X["shape"][0], params["k"],
-                      inds["data"][0], dists["data"][0])
+        m._kneighbors(X["data"][0], X["shape"][0],
```
Comment (JohnZed) [inline dask_cuml/neighbors/nearest_neighbors.py:203]: I like the new order better!

**Review by JohnZed — DISMISSED** [review]
LGTM.

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
This is strange. It looks like 2 of the tests running in CI were giving old errors (pre-PR)

**Comment by cjnolet** [comment]
It looks like this is installing version 0.7 of all the RAPIDS packages when testing, which is causing the tests to fail. 

@dantegd @raydouglass @dillon-cullinan, I notice an `update_version.sh` script in the ci directory. Would updating the version to 0.8 cause CI to install the 0.8 conda packages for the RAPIDS projects?

**Comment by raydouglass** [comment]
> It looks like this is installing version 0.7 of all the RAPIDS packages when testing, which is causing the tests to fail.
> 
> @dantegd @raydouglass @dillon-cullinan, I notice an `update_version.sh` script in the ci directory. Would updating the version to 0.8 cause CI to install the 0.8 conda packages for the RAPIDS projects?

This line in the gpu test scripts controls which versions are installed: https://github.com/rapidsai/dask-cuml/blob/branch-0.8/ci/gpu/build.sh#L35

Since there is not specific version requests and `rapidsai` channel is before `rapidsai-nightly`, conda will find and use v0.7 packages instead of v0.8. You can either make the version explicit or change the order of channels.

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
Build is still failing from the upstream `LocalCUDACluster` issue but the good news is we have resolved the centos build problem.

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
rerun tests

**Pull request merged**